demos/image-classification/index.js

```diff
@@ -181,6 +183,18 @@ const main = async () => {
     }
   } catch (err) {
     log(`[Error] ${err}`);
+
+    status.setAttribute("class", "red");
+    info.innerHTML = `
+            ${error}<br>
+            Your device probably doesn't have a supported GPU.`;
```

Comment:
```suggestion
            Your device probably doesn't have a supported GPU/NPU.`;
```
Is this code path exclusive to GPU? If so, ignore my suggestion.

Reply:
From what I could tell in the code + from testing, yes that path is GPU specific